view/frontend/web/js/view/payment/method-renderer/pstk_paystack.js

```diff
@@ -105,19 +105,28 @@ define([
         callback: function(response) {
           $.ajax({
             method: "GET",
-            url:
+            
+            /*url:
               paystackConfiguration.api_url +
               "paystack/verify/" +
               response.reference +
-              "_-~-_" +
-              quoteId
+              "_-~-_" + 
+              quoteId */ 
+              url:
+              "https://api.paystack.co/transaction/verify/"+response.reference,
+              beforeSend: function(xhr) {
+             xhr.setRequestHeader("Authorization", "Bearer sk_test_7e06e495c40565daef701e95b3dfeefc88541037");
```

Comment:
@thetechtitan your secret key cannot be in the frontend. It's supposed to be 'secret' - not viewable by anyone. All API calls are done on the backend.